src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
```

Comment:
The default compare function may not work as expected for numeric sorting since it converts numbers to strings before comparing. Consider using a numeric comparison for numbers to ensure correct sorting order.